bin/datatool/src/args.rs

```diff
@@ -172,6 +185,15 @@ pub(crate) struct SubcParams {
         description = "permit blank proofs after timeout in millis (default strict)"
     )]
     pub(crate) proof_timeout: Option<u32>,
+
+    #[argh(option, description = "fixed fee for writer transaction")]
+    pub(crate) fixed_fee: Option<u64>,
+
```

Comment:
I guess this is more descriptive.
```suggestion
    #[argh(option, description = "Fee in sats for reveal transactions. Will be determined dynamically if None")]
    pub(crate) reveal_fee: Option<u64>,
```